Synology.py

```diff
@@ -198,7 +198,7 @@ def validate():
                     try:
                         run_tool()
                     except Exception as e:
-                        elogger.error(e)
+                        elogger.error(e, exc_info=e)
```

Comment:
This change will have it print a stack trace to the log file rather than just the error message.